db/data_migration/20180607142603_rename_regulatory_judgement_publication.rb

```diff
@@ -0,0 +1,4 @@
+DataHygiene::DocumentReslugger.call(
+  from: "regulatory-judgement-symphony-housing-group-ltd",
+  to: "regulatory-judgement-onwards-homes-limited",
+)
```

Comment:
Have you considered running the document reslugging rake task?
```
rake reslug:document[regulatory-judgement-symphony-housing-group-ltd,regulatory-judgement-onwards-homes-limited]
```

Reply:
Well... I didn't even know that existed! It feels like these reslugging tasks should be better documented.

Reply:
Ha, probably :)